openreview/venue_request/process/matching_pre_process.py

```diff
@@ -20,8 +20,9 @@ def process(client, note, invitation):
             raise openreview.OpenReviewException('Paper matching is already being run for this group. Please wait for a status reply in the forum.')
 
     if forum_note.content.get('api_version', '1') == '2':
-        senior_area_chairs_name = domain.get_content_value('senior_area_chairs_name')
-        if senior_area_chairs_name and matching_group.endswith(senior_area_chairs_name) and forum_note.content.get('senior_area_chairs_assignment', 'Area Chairs') == 'Area Chairs':
+        senior_area_chairs_id = domain.get_content_value('senior_area_chairs_id')
+        senior_area_chairs_assignment = forum_note.content.get('senior_area_chairs_assignment', 'Area Chairs')
```

Comment:
given that you have the domain group, you can check the variable `sac_paper_assignments` instead of the string "Area Chairs". 

`if not domain.get_content_value('sac_paper_assignments')`

Reply:
Didn't know of that variable, fixed.